Source/WTF/wtf/URL.cpp

```diff
@@ -1307,15 +1307,15 @@ bool isEqualIgnoringQueryAndFragments(const URL& a, const URL& b)
     return substringIgnoringQueryAndFragments(a) == substringIgnoringQueryAndFragments(b);
 }
 
-void removeQueryParameters(URL& url, const HashSet<String>& keysToRemove)
+Vector<String> removeQueryParameters(URL& url, const HashSet<String>& keysToRemove)
```

Comment:
Hopefully this isn't used in performance sensitive code paths.

Reply:
Hm...not as far as I know (AFAIK, in all the contexts where this helper method is used, appending some strings to a `Vector` should be very cheap, compared to the rest of the work those contexts). If this shows up in traces, we can surely add a different version of this that doesn't return anything, however.